src/server_manager/cloud/gcp_api.ts

```diff
@@ -108,47 +120,70 @@ function instanceUrl(instance: InstanceLocator): string {
  * @see https://cloud.google.com/compute/docs/reference/rest/v1/instances/getGuestAttributes#response-body
  */
 type GuestAttributes = Readonly<{
-  variableKey: string; variableValue: string; queryPath: string;
-  queryValue: {items: Array<{namespace: string; key: string; value: string;}>;};
+  variableKey: string;
+  variableValue: string;
+  queryPath: string;
+  queryValue: {items: Array<{namespace: string; key: string; value: string}>};
 }>;
 
 /** @see https://cloud.google.com/compute/docs/reference/rest/v1/zones */
 type Zone = Readonly<{
-  id: string; creationTimestamp: string; name: string; description: string; status: 'UP' | 'DOWN';
+  id: string;
+  creationTimestamp: string;
+  name: string;
+  description: string;
+  status: 'UP' | 'DOWN';
   region: string;
 }>;
 
 type Status = Readonly<{code: number; message: string}>;
 
 /** @see https://cloud.google.com/resource-manager/reference/rest/Shared.Types/Operation */
-export type ResourceManagerOperation = Readonly<{name: string; done: boolean; error: Status;}>;
+export type ResourceManagerOperation = Readonly<{name: string; done: boolean; error: Status}>;
 
 /**
  * @see https://cloud.google.com/compute/docs/reference/rest/v1/globalOperations
  * @see https://cloud.google.com/compute/docs/reference/rest/v1/zoneOperations
  */
-export type ComputeEngineOperation = Readonly<
-    {id: string; name: string; targetId: string; status: string; error: {errors: Status[]}}>;
+export type ComputeEngineOperation = Readonly<{
+  id: string;
+  name: string;
+  targetId: string;
+  status: string;
+  error: {errors: Status[]};
+}>;
 
 /**
  * @see https://cloud.google.com/service-usage/docs/reference/rest/Shared.Types/ListOperationsResponse#Operation
  */
-type ServiceUsageOperation = Readonly<{name: string; done: boolean; error: Status;}>;
+type ServiceUsageOperation = Readonly<{name: string; done: boolean; error: Status}>;
```

Comment:
Ugh, I don't know why. Maybe it's because it's a oneliner? I'd rather not fight with the formatter. The idea is that we don't have to think about format.